src/test/java/net/sf/jabref/logic/formatter/bibtexfields/HtmlToUnicodeFormatterTest.java

```diff
@@ -0,0 +1,59 @@
+package net.sf.jabref.logic.formatter.bibtexfields;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class HtmlToUnicodeFormatterTest {
+
+    private final HtmlToUnicodeFormatter formatter = new HtmlToUnicodeFormatter();
+
+    @Before
+    public void setUp() throws Exception {
+        Globals.prefs = JabRefPreferences.getInstance();
+    }
+
+    @Test
+    public void formatWithoutHtmlCharactersReturnsSameString() {
+        assertEquals("abc", formatter.format("abc"));
+    }
+
+    @Test
+    public void formatMultipleHtmlCharacters() {
+        assertEquals("åäö", formatter.format("&aring;&auml;&ouml;"));
+    }
+
+    @Test
+    public void formatCombinedAccent() {
+        assertEquals("í", formatter.format("i&#x301;"));
+    }
+
+    @Test
+    public void testBasic() {
+        assertEquals("aaa", formatter.format("aaa"));
+    }
+
+    @Test
+    public void testHTML() {
```

Comment:
I'll separate it in two tests. One that tests for the different variants of `ä` (I think it makes sense to bundle those) and one that tests for the greek letter.